### ISSUE TITLE
I can't get to vst ui by pressing f6 in fx window

### ISSUE THREAD

**Issue by c469591** [jcsteh/osara#792]
> 

Hello.
I recently found a problem when I was using an old version of osara, about before the pre990 version.
When I loaded vst on the track, I could open the fx window and press f6 to enter the vst plugin interface.
For example, I loaded Ozone 9, after I pressed f6, then pressed tab, I could see a button like this →
Preset Readout Frame button Enter
Options button Enter
Left Arrow button Enter
Right Arrow button Enter
History button Enter
Help button Enter
Then I can press enter or the arrow keys to control these functions.
But recently I updated osara, using the pre996 version, also have tried to download the latest version of osara, press f6 are not working, will only switch the solo state of the track.
I used the installed version of reaper and the portable version of reaper, both versions are 6.67, only the difference of osara version, and found that the old version of osara is normal, but the new version of osara can not enter the plug-in ui by pressing f6.
Is there a new way to do it? Or is this a problem with osara?
Thanks!

Translated with www.DeepL.com/Translator (free version)

**Comment by RDMurray** [comment]
Is this only a problem with certain plugins? Please can you test with one of Reaper's built-in effects such as ReaEq. I I know it isn't needed for ReaEq, but it should still work. 

I don't have Ozone, and I can't reproduce the problem with any of the plugins I do have.

**Comment by c469591** [comment]
hi.
Unfortunately, reaper's own vst can be directly in the fx window, press tab to see all the options, no need to press f6.
In fact, it is not necessary to test Ozone 9, almost any other vst that is not reaper's original can be used for testing.
For example, Melodyne or izotope rx9 vst version can be used for testing.
In the past, as long as the plug-in is loaded, press f6 in the fx window to move the focus of nvda to the plug-in interface.
The difference is in the plug-in interface after entering the tab, some plug-ins can see the plug-in buttons, sliders, etc., some press the tab is nothing.
But one thing is the same, that is, after pressing f6 will enter the plug-in interface.
And now, I use the latest version of osara, press f6 can only switch the solo state of the track, can not move the focus of nvda to the plug-in interface .
I hope my description can let you understand, if there is not understand the place I can continue to add.
Thank you!



Translated with www.DeepL.com/Translator (free version)

**Comment by Lo-lo78** [comment]
@c469591 
Give it a try with a portable Reaper.
This way you will have the clean config folder and you will see that OSARA worksbene.
I haven't encountered any of the problems you mentioned.
There are plugins that don't work with F6. But if you're on the FX chain it doesn't just go into the track.
There is probably some problem in your Reaper config folder that isn't working.

**Comment by RDMurray** [comment]
@c469591  The code related to that function hasn't changed in a long time, so I'm no closer to solving your problem unfortunately.  

I know it is not needed for ReaEq, but it should still work for ReaEq.

Please try adding ReaEq to a track, then in the fx list when focused on ReaEq, press F6.  Does it say solo? It should move to the band frequency control.

Also, what is the window title when in the fx window?  You can copy it to the clipboard with NVDA+T 3 times.

**Comment by ScottChesworth** [comment]
Hey @c469591, are you sure that the effect isn't being run in bridged mode? That would break the functionality on F6.

**Comment by c469591** [comment]
Hi, guys.
I was able to find the cause of the problem.
I just switched to English and reopened reaper and did a test and everything is fine.
I also found that the old version of osara's reaper is also using the old reaper language pack.
So I think I should go to the person who made the new version of the traditional Chinese language pack.
But honestly, I can't imagine that the problem is caused by the language pack.
Thank

**Comment by ScottChesworth** [comment]
I'm gonna close this issue because it doesn't seem like there's anything needed from the OSARA team right now. If you find that we can improve this situation when you've communicated with the person who makes the language pack, feel free to reopen it, come back to us with more details.

**Comment by RDMurray** [comment]
I thought it might be the language pack. OSARA uses the window title to detect the fx window. 

I think this is still a bug in OSARA. Given that we support translation now, we should really find a better way to detect the fx window. 

@c469591 this isn't the fault of the language pack author, but deleting the translation for "FX: " in the language pack or reverting to the earlier version would be a temporary workaround. 

Reopening.

**Comment by RDMurray** [comment]
This would also break the announcement of active or bypassed in the FX list.

**Comment by c469591** [comment]
I understand, I'm looking forward to the future osara to solve this problem.
I contacted the translator yesterday, but he didn't think it was a problem with the translation package, so I had a feeling of helplessness that I couldn't start.
Luckily, I saw the reply here today.
Thank you!

**Comment by FelipeZanabria** [comment]
I use Reaper in Spanish, and I confirm the problem presented here.
I thought that the f6 shortcut didn't work because of some plugin without an accessible gui, that's why I didn't report it.
I've done the test with reaEq, and indeed, it solos the track.
In the language pack there is a section called like this:
[FX]
I found with notepad the FX chain:
It was written without a space after :
When placed, it works normally.
I noticed a missing space setting Reaper in English.
I know that an object has different properties, so I confirm that another property of the fx window should be chosen.
